### PR TITLE
ZCS-816 Conditional ivy settings for network build

### PR DESCRIPTION
**Pull request by gordyt**
If the `zm-network-build` repo is checked-out locally, then
reference the `ivysettings-network.xml` (which includes our
internal maven repository).  If not, reference
`ivysettings.xml`.

This plus another ZCS-816 pull request for `zm-network-build` fixes network builds

### CODE REVIEW

File: ant-global.xml [hunk 1]
```diff
@@ -38,7 +38,15 @@
             <fileset dir="${ivy.jar.dir}" includes="*.jar"/>
         </path>
         <taskdef resource="org/apache/ivy/ant/antlib.xml" uri="antlib:org.apache.ivy.ant" classpathref="ivy.lib.path"/>
-        <ivy:settings id="dev.settings" file="../zm-zcs/ivysettings.xml"/>
+        <if>
+          <available file="../zm-network-build" type="dir"/>
+          <then>
+            <ivy:settings id="dev.settings" file="../zm-zcs/ivysettings-network.xml"/>
```
Comment (grishick) [inline ant-global.xml]: shouldn't ivysettings-network.xml be in zm-network-build folder?

Comment (gordyt) [reply ant-global.xml]: That is an excellent point.  It made sense at the time that Prashant and I were talking about it.  I will relocate the file and update the two PRs.

Comment (gordyt) [reply ant-global.xml]: Greg I have updated both PRs and am re-running a network build now to test.

Comment (gordyt) [reply ant-global.xml]: Verified new network build with changes you suggested.

**Review by grishick — APPROVED** [review]

**Review by silentsakky — APPROVED** [review]

**Pull request merged**